python-avd/pyavd/_eos_designs/schema/schema_fragments/design.schema.yml

```
@@ -11,17 +11,7 @@ keys:
       table: design
     deprecation:
       warning: true
-      removed: false
+      removed: true
       remove_in_version: 6.0.0
-      url: https://avd.arista.com/5.x/docs/porting-guides/5.x.x.html#deprecation-of-designtype
+      url: https://avd.arista.com/5.x/docs/porting-guides/6.x.x.html#removal-of-designtype
```

Let's do it like this for now it will be broken but with #5903 it will be ignored for now

```suggestion
      url: https://avd.arista.com/6.x/docs/porting-guides/6.x.x.html#removal-of-designtype
```